src/content/api/compilation-hooks.md

```diff
@@ -15,7 +16,7 @@ During the compilation phase, modules are loaded, sealed, optimized, chunked,
 hashed and restored.
 
 The `Compilation` class also extends `Tapable` and provides the following
-lifecycle hooks. They can be tapped the same way as compiler hooks:
+life-cycle hooks. They can be tapped the same way as compiler hooks:
```

Comment:
I think this can also be `life cycle` and `lifecycle`. Why updating it now?

Reply:
I just felt that life-cycle is better.

Reply:
Same goes for life-cycle, i had found few resources on the internet which ensure me that all three versions can be used, and be correct. Unless this is wrong/misused word i would suggest reverting the changes to not change for the sake of change

Reply:
Ok i'll revert it.